Compat for My Nether's Delight Refabricated

wawwior opened the pull request:
https://modrinth.com/mod/my-nethers-delight-refabricated
Tested to work on fabric ~~the forge version should work too~~ the forge version might work

Also sorry for the auto formatting

wawwior committed:
Compat for My Nether's Delight Refabricated

wawwior committed:
Merge branch 'multiloader-1.20.1' of https://github.com/ThatJadenXgamer/Jadens-Nether-Expansion into multiloader-1.20.1

wawwior committed:
Compat for My Nether's Delight Forge

!! Untested !!

wawwior committed:
Forgot to stage this :3

ThatJadenXgamer reviewed (changes requested):
Make the nether pizza recipe an item tag in the main resource, and add hoglin loin to that tag in the built-in datapack

wawwior committed:
feat: Make Pizza Recipe use Tag

Pizza Recipe uses new Tag '#netherexp:cooked_ham'
Tag is modified in compat datapack instead of recipe itself

wawwior committed:
Merge branch 'multiloader-1.20.1' into nethers-delight-compat
